roles/pulp/templates/pulp.container.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Pulp 
+After=network.target
+
+[Container]
+Image={{ pulp_image }}
+ContainerName={{ pulp_container_name }}
+PublishPort={{ pulp_ports }}
+{% for volume in pulp_volumes %}
+Volume={{ volume }}
+{% endfor %}
+Environment=PULP_CONTENT_ORIGIN=http://{{ ansible_hostname }}:8080
+
+[Service]
+ExecStart=/usr/bin/podman run --name %n --rm --device /dev/fuse {{ pulp_image }}
```

Comment:
This section has to be dropped, quadlet's generate a `ExecStart` and this causes a duplicate which breaks the service.